CourseManager/Views/ExampleViews/MatchedGeometryEffect/Views/Components/FiveStarRatingView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  FiveStarRatingView.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-12-08.
+//
+
+import SwiftUI
+
+struct FiveStarRatingView: View {
+    @State var appear = false
+    @State var rating: Double = 4.6
+    var body: some View {
+        ZStack {
+            starsView
+                .overlay(
+                    overlayView
+                )
+                .mask(starsView)
+        }
+        .onAppear {
+            withAnimation(.spring().delay(0.7)) {
+                appear = true
+            }
+        }
+    }
+
+    private var starsView: some View {
+        HStack(spacing: 0) {
+            ForEach(/*@START_MENU_TOKEN@*/0 ..< 5/*@END_MENU_TOKEN@*/) { _ in
```

Comment:
remove hidden code

Reply:
Not sure why it's showing like that, in my code it's actually:
  ForEach(0 ..< 5) { _ in
                Image(systemName: "star.fill")
                    .foregroundColor(.gray)
            }